packages/webapp/src/components/Crop/PlantingLocation/index.jsx

```diff
@@ -98,11 +100,24 @@ export default function PurePlantingLocation({
   const handlePinMode = () => {
     setPinToggle((pinToggle) => !pinToggle);
   };
+  const crop = useSelector(cropVarietySelector(variety_id));
+
+  const onSubmit = () => {
+    const d = getValues();
+    const selectedLocationId =
+      d?.crop_management_plan?.planting_management_plans[locationPrefix]?.location_id;
+    const selectedLocation = cropLocations.find((c) => c.location_id === selectedLocationId);
+    const isCropOrganic = crop.organic;
+    const isSelectedLocationOrganic = selectedLocation.organic_status === 'Organic';
+    if (isCropOrganic !== isSelectedLocationOrganic) {
```

Comment:
I think the default value for crop.organic could be null in some cases, based the cropVarietyModel in the api. If null the expression is always true. Is there a way to set this property? I haven't found it in the ui.